resources/views/admin_layout.blade.php

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<head>
+<title>Dashboard</title>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+<meta name="keywords" content="Visitors Responsive web template, Bootstrap Web Templates, Flat Web Templates, Android Compatible web template, 
+Smartphone Compatible web template, free webdesigns for Nokia, Samsung, LG, SonyEricsson, Motorola web design" />
+<script type="application/x-javascript"> addEventListener("load", function() { setTimeout(hideURLbar, 0); }, false); function hideURLbar(){ window.scrollTo(0,1); } </script>
+<!-- bootstrap-css -->
+<link rel="stylesheet" href="{{asset('backend/css/bootstrap.min.css')}}" >
+
+<!-- //bootstrap-css -->
+<!-- Custom CSS -->
+<link href="{{asset('backend/css/style.css')}}" rel='stylesheet' type='text/css' />
```

Comment:
```suggestion
<link href="{{ asset('backend/css/style.css') }}" rel='stylesheet' type='text/css' />
```
fix tương tự trên với các chỗ khác